pages/attached_result.html

```diff
@@ -78,8 +78,8 @@
                 <script type="text/html" class="result-template">
                     <div class="coveo-result-frame">
                       <div class="CoveoResultActionsMenu">
-                        <div class="CoveoQuickview"></div>
                         <div class="CoveoAttachResult" data-attach-caption="Attach" data-detach-caption="Detach"></div>
+                        <div class="CoveoQuickview"></div>
```

Comment:
What's the change here?

Reply:
There is a bug caused by the fact the the Quickview Icon is sometimes hidden in the ResultAction Menu. When the Quickview icon is hidden, its neighbor to the right still has a left border, creating a visual glitch. 

Placing the Quickview Icon at the far right of the  ResultAction Menu patches that bug, but does not fix it. 

I will revert that change so it does not hide the bug